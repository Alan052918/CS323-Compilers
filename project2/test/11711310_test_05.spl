struct Type {
  char name[32];
};

struct Type {
  int size;
};

int main() {
  struct Type t;
}

