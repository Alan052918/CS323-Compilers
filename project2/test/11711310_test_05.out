Error type 15 at Line 5: redefine struct: Type
